aws/validators.go

```diff
@@ -897,6 +897,8 @@ func validateCloudFormationTemplate(v interface{}, k string) (ws []string, error
 	return
 }
 
+var validateOpenAPISpecification = validateCloudFormationTemplate
```

Comment:
As for `diff_suppress_funcs.go`, I suggest renaming `validateCloudFormationTemplate` to `validateJsonOrYaml`.